packages/vitest/src/node/error.ts

```diff
@@ -64,14 +58,13 @@ export async function printError(error: unknown, ctx: Vitest, options: PrintErro
     ctx.logger.error(c.yellow(e.frame))
   }
   else {
-    printStack(ctx, stacks, nearest, errorProperties, (s, pos) => {
+    printStack(ctx, stacks, nearest, errorProperties, (s) => {
       if (showCodeFrame && s === nearest && nearest) {
-        const file = fileFromParsedStack(nearest)
         // could point to non-existing original file
         // for example, when there is a source map file, but no source in node_modules
-        if (nearest.file === file || existsSync(file)) {
-          const sourceCode = readFileSync(file, 'utf-8')
-          ctx.logger.error(c.yellow(generateCodeFrame(sourceCode, 4, pos)))
+        if (existsSync(nearest.file)) {
```

Comment:
This check can also be removed, because it's already done on line 45. It was redone again in the previous iteration because of `fileFromParsedStack`

Reply:
Oh nice catch! will remove 👍

Reply:
Done ✅!